src/morse/api.clj

```diff
@@ -101,6 +101,18 @@
                                :form-params  query})]
      (-> resp :body))))
 
+(defn edit-reply-markup
```

Comment:
The fn have to be renamed `edit-message-reply-markup` to precisely match the Bot API method name.